teamcity: failed test: decommission

The following tests appear to have failed on master (roachtest): acceptance/decommission

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+decommission).

[#1516454](https://teamcity.cockroachdb.com/viewLog.html?buildId=1516454):


```
acceptance/decommission
--- FAIL: roachtest/acceptance/decommission (12.516s)
test artifacts and logs in: artifacts/acceptance/decommission/run_1
	decommission.go:505,acceptance.go:69,test_runner.go:689: csv input:
		
		id,is_live,replicas,is_decommissioning,is_draining
		3,true,0,true,false
		
		No more data reported on target nodes. Please verify cluster health before removing the nodes.
		
		expected:
		[][]string{
		    {"id", "is_live", "replicas", "is_decommissioning", "is_draining"},
		    {"3", "true", "0", "true", "true"},
		    {"No more data reported on target nodes. Please verify cluster health before removing the nodes."},
		}
		errors:<nil>
		row #2, col #5: found "false" which does not match "true"

test artifacts and logs in: artifacts/acceptance/decommission/run_1
	decommission.go:505,acceptance.go:69,test_runner.go:689: csv input:
		
		id,is_live,replicas,is_decommissioning,is_draining
		3,true,0,true,false
		
		No more data reported on target nodes. Please verify cluster health before removing the nodes.
		
		expected:
		[][]string{
		    {"id", "is_live", "replicas", "is_decommissioning", "is_draining"},
		    {"3", "true", "0", "true", "true"},
		    {"No more data reported on target nodes. Please verify cluster health before removing the nodes."},
		}
		errors:<nil>
		row #2, col #5: found "false" which does not match "true"





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cmd/roachtest/decommission.go
+++ b/pkg/cmd/roachtest/decommission.go
@@ -497,8 +497,11 @@ func runDecommissionAcceptance(ctx context.Context, t *test, c *cluster) {
 		exp := [][]string{
 			decommissionHeader,
 			// Expect the same as usual, except this time the node should be draining
-			// because it shut down cleanly (thanks to `quit --decommission`).
-			{"3", "true", "0", "true", "true"},
+			// because it shut down cleanly (thanks to `quit --decommission`). It turns
+			// out that while it will always manage to mark itself as draining during a
+			// graceful shutdown, gossip may not yet have told this node. It's rare,
+			// but seems to happen (#41249).
+			{"3", "true", "0", "true", "true|false"},
 			decommissionFooter,
 		}
 		if err := matchCSV(o, exp); err != nil {

